README.rst

```diff
@@ -420,18 +317,18 @@ of them, you can specify the number of pages:
     records = client.get_records(_limit=10, pages=float('inf'))  # Infinity
 
 
-Generating endpoint paths
--------------------------
+Endpoint URLs
+-------------
 
-You may want to generate some endpoint paths, you can use the
-get_endpoint utility to do so:
+The ``get_endpoint()`` method returns the endpoint URL on the server:
```

Comment:
I'd probably write `returns an endpoint URL` instead of `returns the endpoint URL`, since there are several endpoints and it can return any of them.